geopandas/geoseries.py

```diff
@@ -751,11 +751,27 @@ def notnull(self):
         return self.notna()
 
     def fillna(self, value=None, method=None, inplace=False, **kwargs):
-        """Fill NA values with a geometry (empty polygon by default).
+        """
+        Fill NA values with geometry (or geometries).
+
+        ``method`` is currently not implemented.
+
+        Parameters
+        ----------
+        value : Geometry or GeoSeries, default None
```

Comment:
```suggestion
        value : scalar geometry or GeoSeries, default None
```
`Geometry` implies a class but given we support multiple engines now, better to generalise it.